src/hipscat/pixel_math/partition_stats.py

```diff
@@ -136,18 +136,19 @@ def generate_destination_pixel_map(histogram, pixel_map):
         dictionary that maps the integer 3-tuple of a pixel at destination order to the set of
         indexes in histogram for the pixels at the original healpix order
     """
-
     non_none_elements = [i for i in pixel_map if i is not None]
     unique_pixels = np.unique(non_none_elements, axis=0)
 
+    order = hp.npix2order(len(histogram))
```

Comment:
It is worth adding a comment here, because I was not sure why you are computing the order from the length of the histogram until I looked at the histogram code above (and the comment that the length of the histogram is equal to the number of healpix at that order).

I added an example, but feel free to do something else.